src/styles/blocks/header.scss

```diff
@@ -0,0 +1,42 @@
+.header {
+  background-image: url(/src/images/backround-img/header-bg-mobile.png);
```

Comment:
[Not critical]
The header image for narrow screens doesn't look well enough - there's too much empty space of the image displayed:
![header320](https://github.com/mate-academy/layout_landing-page/assets/6094468/5f832b19-00fc-4ae0-8395-273cbbe15a87)
I'd suggest to play with media query+background styles to improve this.